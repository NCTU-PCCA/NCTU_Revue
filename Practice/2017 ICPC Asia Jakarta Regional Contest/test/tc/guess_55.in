1060486266347 40
><><><<<<><<<<>>>>><><<<<>><><<<><<><<<<
