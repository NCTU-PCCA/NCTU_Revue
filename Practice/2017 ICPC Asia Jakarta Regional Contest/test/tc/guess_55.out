1 1060486266347 2 1060486266346 3 1060486266345 1060486266344 1060486266343 1060486266342 4 1060486266341 1060486266340 1060486266339 1060486266338 5 6 7 8 9 1060486266337 10 1060486266336 1060486266335 1060486266334 1060486266333 11 12 1060486266332 13 1060486266331 1060486266330 1060486266329 14 1060486266328 1060486266327 15 1060486266326 1060486266325 1060486266324 1060486266323
